src/modlunky2/sprites/tilecode_extras.py

```diff
@@ -93,7 +93,8 @@ class TilecodeExtras(BaseSpriteLoader):
     _chunk_map = {
         tilename: (idx, 0, idx + 1, 1) for idx, tilename in enumerate(TILENAMES)
     }
-
+    _chain_index = TILENAMES.index("chain_ceiling")
```

Comment:
I am not sure about this part, is this an acceptable way of making a temp var for this purpose?

Reply:
I don't really like the overrides like this. maybe update the `"chain_ceiling"` entry to be `["chain_ceiling", "chainandblocks_ceiling"]`. and add a

```py
def _generate_chunk_map():
    chunk_map = {}
    for idx, tilenames in enumerate(TILENAMES):
        if isinstance(tilenames, str):
            tilenames = [tilenames]
        for tilename in tilenames:
            chunk_map[tilename] = (idx, 0, idx+1, 1)
    return chunk_map
```
this lets us easily reuse any index for multiple tile names.

Reply:
That is a good idea, however I decided to just create a new image for the chainandblocks_ceiling instead since that way it can be differentiated from the chain_ceiling.